WikiDrips/WikiDoc.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  WikiDoc.swift
+//  WikiDrips
+//
+//  Created by Bart Cullimore on 4/3/17.
+//  Copyright © 2017 Bart Cullimore. All rights reserved.
+//
+
+import Foundation
+
+public class WikiDoc
+{
+    public let title = "Test Title String"
```

Comment:
Since this string will probably appear in UI, look into using `NSLocalizedString(key: String, comment: String)`